src/null-safety/index.md

```diff
@@ -3,7 +3,10 @@ title: Sound null safety
 description: Information about Dart's null safety feature
 ---
 
-The Dart language comes with sound null safety.
+The Dart language comes with sound null safety. 
+
+In Dart 3, null safety is always enabled. In Dart 2.x,
+it must be enabled [with a pubspec setting](#enable-null-safety).
```

Comment:
Is it possible to _disable_ null safety in Dart 3?
The note you're removing a few lines down says:
> Dart 3--planned for a mid-2023 release--
will _require_ sound null safety. It will prevent code from running without it.

Saying it's "always enabled" makes it sound like it can be disabled.
I was under the impression that it is non-negotiable, and if that's true we should say:
```suggestion
In Dart 3, null safety is required. In Dart 2.x,
it must be enabled [with a pubspec setting](#enable-null-safety).
```